sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -994,7 +1069,13 @@ def input_for(transform_id, input_id):
         # The worker will be waiting on these inputs as well.
         for other_input in data_input:
           if other_input not in deferred_inputs:
-            deferred_inputs[other_input] = _ListBuffer([])
+            outputs = process_bundle_descriptor.transforms[
+              other_input].outputs.values()
+            coder_id = process_bundle_descriptor.pcollections[
+              only_element(outputs)].coder_id
+            coder = context.coders[coder_id]
+            deferred_inputs[other_input] = _ListBuffer(
+                coder_impl=coder.get_impl())
```

Comment:
As commented at L1082 (of the PR branch), deferred inputs cannot be parallel processed for now. Is it better to set coder_impl to None to reduce unnecessary processes for now and add it back later when parallel processing is supported for deferred_inputs?